app/Types.hs

```diff
@@ -1,12 +1,19 @@
-{-# LANGUAGE DeriveAnyClass,
```

Comment:
I'm not sure exactly _how_, but one of these extension changes is causing the issue with deleting TODO's. When I tried this locally, the issue was that the server expects the delete request to be simply an array of ints, because `DeleteTodoRequest` is a newtype and the FromJSON is being automatically derived. 

The quick fix: not deriving `FromJSON` and instead declared the instance separately, things started working. ie 

```
newtype DeleteTodoRequest = DeleteTodoRequest
  { ids :: [Integer]
  } deriving (Show, Generic)
instance FromJSON DeleteTodoRequest
```

Reply:
pushed a fix.  Thank you @aviaviavi!